src/app/pages/acta-recibido/edicion-acta-recibido/edicion-acta-recibido.component.scss

```
@@ -65,3 +65,10 @@ table.consolidado {
 p {
   margin-top: 0.5rem;
 }
+
+:host /deep/ .completer-dropdown {
```

Cambiar `/deep/` por `::ng-deep` para facilitar una inminente migracion a Angular 8, aunque también está [mandada a recoger](https://dev.to/jwp/angular-10-ng-deep-and-host-pseudo-classes-1kbg)

```suggestion
:host ::ng-deep .completer-dropdown {
```

(**Alternativamente/mejor**: mover este tipo de estilos a un punto más general, afuera, para no tener que agregarle el deep)